### ISSUE TITLE
Cannot Import Pypath

### ISSUE THREAD

**Issue by cheys08** [saezlab/pypath#240]
Once pypath is installed, I attempted to import it to be able to access omnipath. I am met with 'no module named pypath' Also when I install omnipath, it states that there is an error with building the wheel. 

`pip install pypath`
`pip install pypath-omnipath`

I expect for there to be no error and for me to be able to import the databases without error. 

`Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 10
  ╰─> [1 lines of output]
      Please specify --curl-dir=/path/to/built/libcurl
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 10
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.`

 - OS: HP
 - Python version: 3.10.11

**Comment by deeenes** [comment]
As the error message suggests, the `libcurl` shared objects are not in the `LIBRARY_PATH` configured for your compiler. The most likely reasons are `libcurl` not being installed on your system, or it is installed to a non-standard location, or the paths are not configured correctly. When you try to address this issue, you can test your solution by installing `pycurl` only, as the issue is not directly related to `pypath`. If you have difficulties finding a solution or have any question, feel free to ask here with providing the relevant information.

Btw, `pip install pypath` installs a different package from someone else, to install this package use `pip install pypath-omnipath` or even better `pip install git+https://github.com/saezlab/pypath.git`. If you only want to access OmniPath data, I recommend using the [client package](https://github.com/saezlab/omnipath) instead.

**Comment by cheys08** [comment]
I attempted to install the package 'pip install pypath-omnipath'
and I am met with this error:
PS C:\Users\chey0> pip install pypath-omnipath
Collecting pypath-omnipath
  Using cached pypath_omnipath-0.14.48-py3-none-any.whl (1.1 MB)
Collecting PyYAML (from pypath-omnipath)
  Downloading PyYAML-6.0-cp311-cp311-win_amd64.whl (143 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 143.2/143.2 kB 2.1 MB/s eta 0:00:00
Collecting beautifulsoup4 (from pypath-omnipath)
  Using cached beautifulsoup4-4.12.2-py3-none-any.whl (142 kB)
Collecting dill (from pypath-omnipath)
  Using cached dill-0.3.6-py3-none-any.whl (110 kB)
Collecting future (from pypath-omnipath)
  Using cached future-0.18.3.tar.gz (840 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting glom (from pypath-omnipath)
  Using cached glom-23.3.0-py3-none-any.whl (101 kB)
Collecting lxml (from pypath-omnipath)
  Downloading lxml-4.9.2-cp311-cp311-win_amd64.whl (3.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.8/3.8 MB 3.8 MB/s eta 0:00:00
Collecting matplotlib (from pypath-omnipath)
  Downloading matplotlib-3.7.1-cp311-cp311-win_amd64.whl (7.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 7.6/7.6 MB 5.1 MB/s eta 0:00:00
Collecting numpy (from pypath-omnipath)
  Downloading numpy-1.24.3-cp311-cp311-win_amd64.whl (14.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 14.8/14.8 MB 5.6 MB/s eta 0:00:00
Collecting openpyxl (from pypath-omnipath)
  Using cached openpyxl-3.1.2-py2.py3-none-any.whl (249 kB)
Collecting pandas (from pypath-omnipath)
  Downloading pandas-2.0.2-cp311-cp311-win_amd64.whl (10.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.6/10.6 MB 5.7 MB/s eta 0:00:00
Collecting psutil (from pypath-omnipath)
  Using cached psutil-5.9.5-cp36-abi3-win_amd64.whl (255 kB)
Collecting pycurl (from pypath-omnipath)
  Using cached pycurl-7.45.2.tar.gz (234 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 10
  ╰─> [1 lines of output]
      Please specify --curl-dir=/path/to/built/libcurl
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 10
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

It seems like there is an issue with the wheel. I tried installing pycurl before installing 'pip install pypath-omnipath' with 'pip install pycurl' I am still met with an error regarding the wheel: 

PS C:\Users\chey0> pip install pycurl
Collecting pycurl
  Using cached pycurl-7.45.2.tar.gz (234 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 10
  ╰─> [1 lines of output]
      Please specify --curl-dir=/path/to/built/libcurl
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 10
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

I updated the python I am using in hopes it would solve the issue.
OS: HP
Python: 3.11.3

 However, I am still met with the same error. I do have libcurl installed as it was an extension with the installation of git. So, I do not know why I am having an issue with the pycurl.
